react-app/src/native/components/leagues/Schedule.js

```diff
@@ -59,41 +47,23 @@ export default class Schedule extends React.Component {
             teamTable.data.push([team.numInLeague, team.name]);
         });
 
+
```

Comment:
I can't put a review on line 38 but leagueName shouldn't be getting passed to this screen as a prop, you can get the league name from the league object itself and I believe I made a helper method for getting the formatted league name in LeagueHelpers